examples/sllm_store/save_vllm_model.py

```diff
@@ -98,17 +101,37 @@ def _run_writer(input_dir, model_name):
     required=True,
     help="Model name from HuggingFace model hub.",
 )
+parser.add_argument(
+    "--local_model_path",
+    type=str,
+    required=False,
+    help="Local path to the model snapshot.",
+)
 parser.add_argument(
     "--storage_path",
     type=str,
     default="./models",
     help="Local path to save the model.",
 )
+parser.add_argument(
+    "--tensor_parallel_size",
```

Comment:
Let's use a dash (-) consistently instead of an underscore (_).

Reply:
I think using underscore here is more consistent? It seems that all arguments about sllm store is in underscore? i.e., `mem_pool_size` and `storage_path`.